src/config.js

```diff
@@ -18,18 +18,20 @@ kaya.  If not, see <http://www.gnu.org/licenses/>.
 /* Configuration file */
 /* Feel free to add more things to this file that will help you in development */
 
+const packagejs = require('../package.json');
+
 module.exports = {
   port: 4200,
-  version: '0.2.0',
-  dataPath: 'data/',
-  savedFilesDir : 'saved/',
+  version: packagejs.version,
+  dataPath: '../data/',
+  savedFilesDir : '../saved/',
 
   // blockchain specific configuration
   blockchain: {
     // sets timer for the block confirmation
     blockInterval: 10000, // 10000 : 10 seconds for one block
     blockStart: 0,
-    gasPrice: 100, // Min Gas is now 100
+    minimumGasPrice: "100", // Min Gas is now 100
```

Comment:
This will most likely have to be multiplied by some power of 10, as we have now introduced denominations. The numbers in `gasPrice`, `gasLimit`, `amount`, and so on, are all in the smallest unit.